### DISCUSSION TITLE
Not Sure what I'm doing wrong

### DISCUSSION THREAD

**Discussion by JimJimmy93** [351ELEC/351ELEC#976] [Questions & Answers]
i just got the RG35XX Plus handheld. works great with all pre installed roms. I tried to Install a GBA Rom for a Pokémon variant game. The game loads, but on loading, a Small box appears. one all in Mandarin the other saying Deeplay-Keys(1/1) and then some more mandarin. as the games loads though. i have no button inputs. the direction keys work. the menu button also works, but all options are also mandarin and I cant access them anyway. 

where have i gone wrong haha

thanks to anyone who's willing to assist me

**Comment by dhwz** [comment]
I'm confused, you got a RG35XX Plus running AmberELEC?!?
We don't support that device. So I can't answer your question.

**Reply by JimJimmy93** [reply]
well that's embarrassing, I'm in the wrong place, sorry will delete and leave
